config/releases.exs

```diff
@@ -28,3 +28,8 @@ config :state_mediator, Realtime,
 config :state_mediator, State.Prediction, source: System.get_env("MBTA_TRIP_SOURCE")
 
 config :state_mediator, State.Vehicle, source: System.get_env("MBTA_VEHICLE_SOURCE")
+
+config :api_secrets,
+  firebase_credentials: System.fetch_env!("CR_CROWDING_FIREBASE_CREDENTIALS")
+  secret_key_base: System.fetch_env!("SECRET_KEY_BASE")
+  signing_salt: System.fetch_env!("SIGNING_SALT")
```

Comment:
There should be a newline at the end of every file. (That's what the red circle in the diff here means; the lack of a newline.) You can configure your editor to automatically ensure that when saving. In VSCode, for example, if you open the Settings (cmd-comma) and then search for "newline" you'll see `Files: Insert Final Newline` to check.